docker/py3-native/Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM demisto/py3-tools-ubi:1.0.0.40055
+
+COPY requirements.txt .
+COPY chromium/download_chromedriver.sh .
+COPY repos /repos
+
+# handles chromium + tesseract
+# for tesseract - followed instructions from here: https://computingforgeeks.com/install-tesseract-ocr-on-rocky-almalinux/
+RUN dnf update --nodocs -y  \
+    && dnf upgrade -y --nodocs \
+    && dnf install --nodocs -y wget git \
+    && wget https://dl.google.com/linux/linux_signing_key.pub \
+    && wget https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official \
+    && rpm --import linux_signing_key.pub \
+    && rpm --import RPM-GPG-KEY-CentOS-Official \
+    && mv repos/* /etc/yum.repos.d/ \
+    && dnf install -y --nodocs python3-devel gcc gcc-c++ make wget git unzip llvm-libs libXpm \
+    && dnf install -y --nodocs --enablerepo=rocky8-base --enablerepo=rocky8-release tigervnc-server-minimal xorg-x11-utils google-chrome-stable \
+    && dnf install -y --nodocs --enablerepo=rocky8-base --enablerepo=rocky8-release --enablerepo=fedora-epel ImageMagick \
+    && ./download_chromedriver.sh \
+    && unzip chromedriver_linux64.zip \
+    && mv chromedriver /usr/bin/chromedriver \
+    && rm -rf chromedriver_linux64.zip \
+    && google-chrome --version \
+    && chromedriver --version \
+    && dnf install -y --nodocs git automake make autoconf libtool clang zlib zlib-devel libjpeg libjpeg-devel libwebp libwebp-devel libtiff libtiff-devel libpng libpng-devel pango giflib \
+    && dnf install -y https://vault.centos.org/centos/8/PowerTools/x86_64/os/Packages/giflib-devel-5.1.4-3.el8.x86_64.rpm \
+    && git clone --depth 1 https://github.com/DanBloomberg/leptonica.git \
```

Comment:
I think we should use a tag here to install and not from master. Probably use the latest tag of 1.82.0

Reply:
agreed, will update it tomorrow